tests/rptest/tests/acls_test.py

```diff
@@ -90,7 +96,8 @@ def prepare_cluster(self,
                 [self.security.principal_mapping_rules]
             })
 
-        self.redpanda.set_security_settings(self.security)
+        if reset_security_settings:
```

Comment:
Clean this up please. No need to have duplicates.

Reply:
Reduced duplicates to two. One for pre-start and one for post-start of the cluster.